GameServerLib/Server.cs

```diff
@@ -23,8 +23,11 @@ public Server(Game game, ushort port, string configJson, string blowfishKey)
             _logger = LoggerProvider.GetLogger();
```

Comment:
We also need to remove the `blowfishKey` parameter that enters to this function and the hardcoded key in GameServerConsole.